### PR TITLE
AQI-3030 Add protobuf support for schedule, schedule type and schedule occurrence.

### PR DESCRIPTION
**Pull request by ChrisMarin-AI**

### CODE REVIEW

**Comment by AQI-DanG** [comment]
This is definitely a breaking change do we have anything currently using these models that would be negatively impacted?

File: proto/one/common/schedule/schedule.proto [hunk 1]
```diff
@@ -7,22 +7,35 @@ import "record_auditinfo.proto";
...
+    
+    // unique identifier of the schedule
     string id = 1;
 
+    // schedule name
     string name = 2;
 
-    google.protobuf.StringValue twinRefId = 3;
+    //schedule description
+    google.protobuf.StringValue Description = 3;
```
Comment (AQI-DanG) [inline proto/one/common/schedule/schedule.proto:18]: field names should be camelCased, also below on line 21

**Review by AQI-DanG — CHANGES_REQUESTED** [review]

**Comment by joezhou-aqi** [comment]
> This is definitely a breaking change do we have anything currently using these models that would be negatively impacted?

no, there is no consumer for this.

**Review by joezhou-aqi — CHANGES_REQUESTED** [review]
claros content file should be updated to include the new schedule_occurence object

File: proto/one/common/schedule/schedule.proto (hunk 1 above)
Comment (ChrisMarin-AI) [reply proto/one/common/schedule/schedule.proto]: I changed the names.

**Review by JonathanRobertson-AQI — APPROVED** [review]
nice job Chris

**Review by joezhou-aqi — APPROVED** [review]

**Review by KentMiller — APPROVED** [review]

**Pull request merged**